docs/Cluster-VictoriaMetrics.md

```diff
@@ -908,6 +908,8 @@ Below is the output for `/path/to/vminsert -help`:
      Auth key for /debug/pprof/* endpoints. It must be passed via authKey query arg. It overrides httpAuth.* settings
   -prevCacheRemovalPercent float
      Items in the previous caches are removed when the percent of requests it serves becomes lower than this value. Higher values reduce memory usage at the cost of higher CPU usage. See also -cacheExpireDuration (default 0.1)
+  -prometheusImportResponseCode int
+     HTTP response code for /api/v1/import/prometheus requests. (default 204)
```

Comment:
I think, we should mention pushgateway. It helps identify possible issues with push-gw integration.

Reply:
It would be better add an ability to customize response code via query arg passed to `/api/v1/import*`. Something like `curl ... /api/v1/import/prometheus?response_code=200`